### PR TITLE
Refs proofing

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: 13-transport.Rmd [hunk 1]
```diff
@@ -225,18 +226,18 @@ names(zones_joined)
...
-Trips outside the study region can be included in regional model by a special destination ID covering any trips that go to a zone not represented in the model [@hollander_transport_2016].
+Trips outside the study region can be included in a regional model by a special destination ID covering any trips that go to a zone not represented in the model [@hollander_transport_2016].
 The data in `bristol_od`, however, simply ignores such trips: it is an 'intra-zonal' model.
 
 In the same way that OD datasets can be aggregated to the zone of origin, they can also be aggregated to provide information about destination zones.
 People tend to gravitate towards central places.
 This explains why the spatial distribution represented in the right panel in Figure \@ref(fig:zones) is relatively uneven, with the most common destination zones concentrated in Bristol city center.
-The result is `zones_od`, which contains a new column reporting the number of trip destinations by any mode, is created as follows:
+The result is `zones_od`, which contains a new column reporting the number of trip destinations by any mode, and it iscreated as follows:
```
Comment (Nowosad) [inline 13-transport.Rmd:240]:
```suggestion
The result is `zones_od`, which contains a new column reporting the number of trip destinations by any mode, and it is created as follows:
```

**Comment by Robinlovelace** [comment]
@Nowosad and @jannes-m please can you review this so we can get it merged? Plan with refs is to remove pesky package names.

**Comment by Robinlovelace** [comment]
But we can finish the references in separate PR, this is mostly focuss on transport chapter.

File: 13-transport.Rmd (hunk 1 above)
Comment (Robinlovelace) [reply 13-transport.Rmd]: Thanks, good fix. I need to improve my regex skills.

**Comment by Nowosad** [comment]
I made one tiny change, but otherwise it is good to go 👍🏻

**Comment by Robinlovelace** [comment]
🚀

**Comment by Robinlovelace** [comment]
Just the refs now. Do you want to have a bash at that @Nowosad, picking up on my start? It seems R package refs are an issue, I don't mind removing them.

**Comment by Robinlovelace** [comment]
And a few more chapters @jannes-m is signed up for in #1115

**Comment by Nowosad** [comment]
@Robinlovelace I think that we should keep the packages references (i.e., to show good citing practices to other people); I made a simple PR at https://github.com/geocompx/geocompr/pull/1133, but more importantly, I edited many references at Zotero's geocompr library. Please export them and do a double check.

**Comment by Robinlovelace** [comment]
Good to see @Nowosad. I will put in a PR to that branch so you can review the changes made by the export.

**Pull request merged**